velox/core/PlanNode.h

```diff
@@ -2386,3 +2386,21 @@ class TopNRowNumberNode : public PlanNode {
 };
 
 } // namespace facebook::velox::core
+
+template <>
```

Comment:
@kgpai we can reduce this implementation to a single line using `format_as` if we keep fmt 9.0 as the minimum. Can you share the fmt version used inside Meta? Thanks.

Reply:
@kgpai @pedroerp What do you think?

Reply:
@majetideepak Fmt version is 9.1.0 .